backend/src/test/java/io/debezium/configserver/util/Infrastructure.java

```diff
@@ -49,7 +49,7 @@ public enum DATABASE {
     private static final Network NETWORK = Network.newNetwork();
 
     private static final KafkaContainer KAFKA_CONTAINER =
-            new KafkaContainer(DockerImageName.parse("confluentinc/cp-kafka:7.2.3"))
+            new KafkaContainer(DockerImageName.parse("quay.io/debezium/kafka:" + DEBEZIUM_CONTAINER_VERSION))
```

Comment:
Makes sense. Added the change.

Reply:
Doesn't this conflict with the goal of reproducible builds because it moves the control of tested version to quay.io and release builds? That way we could get build failures when the code is not updated to a recent version right after a release. I suggest we do what we discussed somewhere else, to read the version with `System.getProperty()`. That way we can make sure all versions are compatible and resolve to the version number that we expect. WDYT @ani-sha @Naros?

Reply:
That's most definitely another alternative @rk3rn3r, I think as long as we can minimize the code changes required when we bump from 2.x to 2.y of core the better.

Reply:
@Naros @rk3rn3r So do we agree upon using `latest` and get this PR merged?

Reply:
I would prefer using the same version as the upstream Debezium version used by Maven, in best case from System.getProperties()?